timesketch/api/v1/resources.py

```diff
@@ -339,11 +342,25 @@ def post(self, sketch_id):
             for timeline in sketch.timelines:
                 tl_colors[timeline.searchindex.index_name] = timeline.color
                 tl_names[timeline.searchindex.index_name] = timeline.name
+
+            histogram = None
+            es_total_count_unfiltered = 0
+            try:
+                buckets = result[u'aggregations'][u'data_type'][u'buckets']
+                histogram = buckets
+                for bucket in buckets:
+                    es_total_count_unfiltered += bucket[u'doc_count']
+            except KeyError:
```

Comment:
See above comment, but the KeyError catch should be the line under the buckets = ... line to reduce scope that is being caught.
The rest of the lines in the try block could be moved into an else block.